discord/raw_models.py

```diff
@@ -127,13 +127,21 @@ class RawReactionActionEvent(_RawReprMixin):
         The guild ID where the reaction got added or removed, if applicable.
     emoji: :class:`PartialEmoji`
         The custom or unicode emoji being used.
+    member: Optional[:class:`dict`]
```

Comment:
Even if it's guild-only, it should be named `user` for consistency. Also it's not dict!